adversarial_workflow/evaluators/discovery.py

```
@@ -143,6 +160,64 @@ def parse_evaluator_yaml(yml_file: Path) -> EvaluatorConfig:
+        elif min_version and not isinstance(min_version, str):
+            raise EvaluatorParseError(
+                f"model_requirement.min_version must be a string, got {type(min_version).__name__}"
+            )
+
+        # Validate optional min_context is integer if present
+        min_context = req_data.get("min_context", 0)
+        if min_context:
+            if isinstance(min_context, bool):
+                raise EvaluatorParseError(
+                    "model_requirement.min_context must be an integer, got bool"
+                )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)